src/tests/multihost/alltests/test_default_debug_level.py

```diff
@@ -0,0 +1,144 @@
+            for index in range(len(log_split)-1):
+                log_single_line = log_split[index]
+                pattern = re.compile(r'0x\d*')
+                debug_str = pattern.search(log_single_line)
+                if debug_str.group() > '0x0040':
+                    assert False
+
+    def test_0002_check_default_level_with_auth(self, multihost,
+                                                backupsssdconf):
+        """
+        :title: default debug logs: Check successful login with default
+         log level doesn't generate any logs
+        :id: f40a7c66-6b5f-4f3c-8fcb-6aa12f415473
+        """
+        section = "domain/%s" % ds_instance_name
+        domain_params = {'debug_level': ''}
+        tools = sssdTools(multihost.client[0])
+        tools.sssd_conf(section, domain_params, action='delete')
+        # stop sssd, delete logs and cache, start sssd
+        tools.clear_sssd_cache()
+        check_log_size = "du -c /var/log/sssd/ | awk '/total/ {print $1}'"
+        blog_size = multihost.client[0].run_command(check_log_size,
+                                                    raiseonerr=False)
+        print("before auth:", blog_size.stdout_text)
+        user = 'foo1@%s' % ds_instance_name
+        # Authenticate user
+        client = pexpect_ssh(multihost.client[0].sys_hostname, user,
+                             'Secret123', debug=False)
+        client.login(login_timeout=30, sync_multiplier=5,
+                     auto_prompt_reset=False)
+        alog_size = multihost.client[0].run_command(check_log_size,
+                                                    raiseonerr=False)
+        print("after auth:", alog_size.stdout_text)
+        if alog_size.stdout_text != blog_size.stdout_text:
+            assert False
+
+    def test_0003_bz1893159(self, multihost, backupsssdconf):
+        """
+        :title: default debug logs: Check default level as 0 and 1
+        :id: 79411fe9-99d6-430b-90fd-3eff1807975f
+        """
+        section = "domain/%s" % ds_instance_name
+        domain_params = {'debug_level': ''}
+        tools = sssdTools(multihost.client[0])
+        tools.sssd_conf(section, domain_params, action='delete')
+        multihost.client[0].service_sssd('stop')
+        chmod_cmd = 'chmod 444 /etc/sssd/sssd.conf'
+        cmd = multihost.client[0].run_command(chmod_cmd, raiseonerr=False)
+        assert cmd.returncode == 0
+        tools.remove_sss_cache('/var/log/sssd')
+        multihost.client[0].run_command('systemctl start sssd',
+                                        raiseonerr=False)
+        slog = '/var/log/sssd/sssd.log'
+        log_str = multihost.client[0].get_file_contents(slog).decode('utf-8')
+        find1 = re.compile(r'0x0020')
+        find2 = re.compile(r'0x0010')
+        restore_sssd = 'chmod 600 /etc/sssd/sssd.conf'
+        multihost.client[0].run_command(restore_sssd, raiseonerr=False)
+        if not find1.search(log_str) and find2.search(log_str):
```

Comment:
Could you please add a comment and maybe some () to make this more readable?

Reply:
done